configuration/Configuration.go

```diff
@@ -193,6 +202,22 @@ func (c *Configuration) LoadFromFile(path string) error {
 		return errors.New("Could not determine server URL")
 	}
 
+	if c.Agento.Includedir != "" {
+		matches, err := filepath.Glob(c.Agento.Includedir + "/*.conf")
+		if err != nil {
+			return err
+		}
+
+		if matches != nil {
```

Comment:
This nil check is not needed. `for ... := range` will treat nil as an empty slice.